.github/workflows/build.yaml

```diff
@@ -74,6 +74,10 @@ jobs:
             echo "DOCKER_TAG=${docker_tag}" >> $GITHUB_OUTPUT
             echo "DOCKER_TAG_PR=${docker_tag_pr}" >> $GITHUB_OUTPUT
           fi
+          echo "DOCKER_TAG=${docker_tag}"
```

Comment:
Why is this print-out needed?

Reply:
It's used for debugging, I consider important to print which docker tags will be pushed in every workflow run.